website/docs/r/cur_report_definition.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "aws"
+page_title: "AWS: aws_cur_report_definition"
+sidebar_current: "docs-aws_cur_report_definition"
```

Comment:
This data source documentation page is missing a sidebar section (`Cost and Usage Report Resources`) and link in `website/aws.erb`. The `sidebar_current` should also match the correct format.